src/main/java/core/basesyntax/AreaSupplier.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaSupplier {

```

Comment:
I think this isn't the best naming 
```suggestion
public interface AreaSupplier {

```